playbooks/azure/leapp-rhui-azure.spec

```
@@ -0,0 +1,64 @@
+# Update name of package
+%global provider    azure
+Name:           leapp-rhui-%{provider}
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        Leapp in-place upgrade cloud specific package
+
+# Provider variable is name of subdirectory that is cloud specific, please
+# refer to Leapp documentation/code for what subdir you should use.
+# Examples: aws, azure
+%global leappfilespath  usr/share/leapp-repository/repositories/system_upgrade/el7toel8/files/rhui
```

Not real blocker in snippet repos, but bett could be use of `%{_datadir}` macro instead of `usr/share`

```suggestion
%global leappfilespath  %{_datadir}/leapp-repository/repositories/system_upgrade/el7toel8/files/rhui
```

Paths in the rest of SPEC file has to corrected in such a case as the macro starts with the slash `/usr/share`.